### PR TITLE
Compatibility with capacitor 3

### PR DESCRIPTION
**Pull request by hermitdemschoenenleben**
this PR makes the native market plugin compatible with capacitor 3. I tested it on ios and android

(my fork is a fork of https://github.com/djonmayer/native-market that did most of the work!)

### CODE REVIEW

**Comment by Mendzyy** [comment]
Hello, @priyankpat 

Bhai aa add karo ne please.

**Review by riderx — APPROVED** [review]

**Comment by riderx** [comment]
Good to me, can it be merge ?

File: package.json [hunk 1]
```diff
@@ -53,9 +56,9 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/capacitor-community/native-market"
+    "url": "https://github.com/djonmayer/native-market"
   },
   "bugs": {
-    "url": "https://github.com/capacitor-community/native-market/issues"
+    "url": "https://github.com/djonmayer/native-market/issues"
```
Comment (raycarter) [inline package.json:62]: this should not be changed. the repo is still under capacitor-community.

**Comment by riderx** [comment]
i did publish this with the fix : https://github.com/riderx/native-market

File: package.json (hunk 1 above)
Comment (hermitdemschoenenleben) [reply package.json]: I pushed a change that reverts the change in package url

**Comment by hermitdemschoenenleben** [comment]
I pushed a change that reverts the change in package url

**Comment by riderx** [comment]
@priyankpat can you have a look ?
The fork is working in our app in ios and Android

**Comment by adamvigneault** [comment]
@priyankpat have you abandoned this project?

**Review by adamvigneault — APPROVED** [review]

**Comment by petrot** [comment]
Guys, any update on this PR? Using a forked version doesn't seem to be safe in our project..

**Comment by raycarter** [comment]
we have moved to @capgo/native-market. @petrot what is your security issue by using a fork?

**Comment by petrot** [comment]
@raycarter thanks for the quick answer :) It's not a real security issue, we just prefer using the official repos.

**Comment by riderx** [comment]
Hey @petrot I totally understand your point, I'm the maker behind Capgo.
And I got very annoyed by the lack of maintain of some plugin even official, which is understandable, goal are not aligned between consumer and creator sometimes.

On my side, I have published 10 plugins and will continue to publish more, and keep them up to date. 
My business Capgo (a Capacitor OTA system), make this sustainable for me.
Make open-source plugin and give to the community is my way of doing marketing for Capgo.

I'm creating also an open collective to make this sustainable for other plugin makers as well.
That a good way to align interest of consumer and maker eventually :)

**Pull request merged**